pack/Dockerfile

```diff
@@ -393,32 +393,37 @@ ENV GPUSTACK_HIGRESS_PLUGIN_DIR=/opt/data/plugins
 
 ## Install Skopeo
 
+ARG GOLANG_INSTALL_MIRROR="https://go.dev/dl"
+ARG GOPROXY="https://proxy.golang.org"
```

Comment:
We have the ARG group at the begining. Should we put these there

Reply:
This Golang-related `ARG` should be placed above the Skopeo installation script, similar to how `PYTHON_VERSION` and `S6_OVERLAY_VERSION` are positioned above the Python installation script.